fix(a11y): final fix for WCAG 1.1.1 image alt should describe image content accurately (#17)
scottrigby/basekamp-next: content/events/freenode.md

title: Freenode
images:
  - src: ircfreenodenet.png

>     alt: World map with purple gradient background showing red vertical bars on continents representing IRC network nodes; text reads irc.freenode.net

Reviewer: ```suggestion
    alt: irc.freenode.net text with purple gradient world map background containing red lines to represent IRC network nodes.
```
